src/resolver/resolve_path.zig

```diff
@@ -2011,7 +2009,7 @@ pub fn lastIndexOfSep(path: []const u8) ?usize {
 
 pub fn lastIndexOfSepT(comptime T: type, path: []const T) ?usize {
     if (comptime !bun.Environment.isWindows) {
-        return strings.lastIndexOfCharT(T, path, '/');
+        return std.mem.lastIndexOfScalar(T, path, '/');
```

Comment:
why change this? they do the same thing right now but we will probably change out this implementation to a libc function later + it insulates us from having to make more code changes when zig std lib randomly updates the signature of this function in 3 months